packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -151,10 +153,36 @@ export class BlockList extends Component {
 			isReadOnly,
 			shouldShowInsertionPointBefore,
 			shouldShowInsertionPointAfter,
+			containerStyle,
+			getBlockAttributes,
 		} = this.props;
 
+		const getVerticalAlignmentRemap = ( alingment ) => {
```

Comment:
I do like you suggest.

One note here. This one here is because `verticalALignment` attribute in Columns on web is  top / center /bottom and I need ti remap to flex prop name. Also there might be better naming for prop `columnContainerStyle` and `containerStyle` WDYT ?